pkg/networkpolicy/controller.go

```diff
@@ -689,6 +689,12 @@ func (c *Controller) syncNFTablesRules(ctx context.Context) error {
 		tx.Flush(&knftables.Chain{
 			Name: chainName,
 		})
+		// IPv6 needs ICMP Neighbor Discovery to work
+		tx.Add(&knftables.Rule{
+			Chain: chainName,
+			Rule: knftables.Concat(
```

Comment:
nit: it's more readable to write all-constant strings as one string (https://github.com/kubernetes-sigs/kube-network-policies/pull/49#discussion_r1680259915):
```go
		tx.Add(&knftables.Rule{
			Chain: chainName,
			Rule: "icmpv6 type { nd-neighbor-solicit, nd-neighbor-advert } accept",
		})
```
(same for other similar cases)